schema/ismrmrd_example_extended.xml

```diff
@@ -233,6 +233,11 @@
         <maximum>0</maximum>
         <center>0</center>
       </set>
+      <user_1>
+         <minimum>0</minimum>
```

Comment:
A little annoyed at the extra space, but we really need a PR that fixes whitespaces for this entire repo